src/main/java/wellnus/atomichabit/command/ListCommand.java

```diff
@@ -94,7 +103,7 @@ public void execute() {
             stringOfHabitsBuilder.append(currentHabitString).append(LINE_SEPARATOR);
             taskNo += 1;
         }
-        int FIRST_CHAR = 0;
+
```

Comment:
For this can you move the FIRST_CHAR back here but rename it to a `firstChar` variable instead? It's best to minimise the scope of variables, especially since FIRST_CHAR wouldn't make sense outside of the `execute()` function